.svglintrc.js

```diff
@@ -70,6 +72,14 @@ module.exports = {
               reporter.error(`Size of <path> must be exactly ${iconSize} in one dimension; the size is currently ${width} x ${height}`);
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "extraneous";
+
+            const rawSVG = $.html();
+            if (!svgRegexp.test(rawSVG)) {
+              reporter.error("Detected unexpected characters outside the <svg>");
```

Comment:
Perhaps for the error:
```suggestion
              reporter.error("Unexpected character(s) detected outside the opening and/or closing <svg> tags");
```

Reply:
Thanks for the suggestion @PeterShaggyNoble, that's much better